broker_legs_test.go

```diff
@@ -0,0 +1,240 @@
+package legs
+
+import (
+	"context"
+	"io/ioutil"
+	"testing"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	dssync "github.com/ipfs/go-datastore/sync"
+	gsimpl "github.com/ipfs/go-graphsync/impl"
+	gsnet "github.com/ipfs/go-graphsync/network"
```

Comment:
Just so I learn the convention, why these imports are grouped together?

Reply:
This was just copy and past from `legs_test.go`.  The only convention I follow is grouing stdlib packages and other packages.